documentation/docs/60-appendix/25-debugging.md

```diff
@@ -43,6 +43,13 @@ Here's an example `launch.json`:
 
 Further reading: <https://code.visualstudio.com/docs/editor/debugging>.
 
+## Other Editors
+
+While it's not feasible to provide guides to debugging svelte in other editors here, we've compiled a few community resources that might help if Visual Studio Code is not your preferred IDE:
```

Comment:
I don't think we need to provide a defensive explanation here, just "if you don't use VS Code, here are other guides that might be useful to you" or something along those lines

Reply:
yup, I definitely have a tendency to yap too much lol
reworded it to be more concise and less defensive